File: MODUL 4 Praktikum Pemrograman/PRAKTIKUM 4 PY/PRAK404-2410817120021-Aulia-Az-Zahra.py
while True:
    print("\nPilih program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit")
    n = int(input("Masukkan Pilihan : "))

    if n < 1 or n > 5:
        print("Input anda salah, silahkan coba lagi") 
        continue
    elif n == 5:
        print("Terimakasih, telah menggunakan kalkulator AULIA AZ ZAHRA")
        break
    
    a = float(input("Masukkan nilai pertama : "))
    b = float(input("Masukkan nilai kedua : "))

    if n == 1:
        Hasil = a + b
        print(f"Hasil penjumlahan antara {a:.2f} dengan {b:.2f} adalah {Hasil:.2f}")
    elif n == 2:
        Hasil = a - b
        print(f"Hasil pengurangan antara {a:.2f} dengan {b:.2f} adalah {Hasil:.2f}")
    elif n == 3:
        Hasil = a * b
        print(f"Hasil perkalian antara {a:.2f} dengan {b:.2f} adalah {Hasil:.2f}")
    elif n == 4:
        Hasil = a / b
        print(f"Hasil pembagian antara {a:.2f} dengan {b:.2f} adalah {Hasil:.2f}")

File: MODUL 4 Praktikum Pemrograman/PRAKTIKUM 4 C/PRAK404-2410817120021-Aulia-Az-Zahra.c
#include <stdio.h>
int main()
{
    int n;
    float a, b;
    while(1)
    {
        printf("\nPilih program\n1. penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit");
        printf("\nMasukkan Pilihan : ");
        scanf("%d", &n);

        if(n < 1 || n > 5) 
        {printf("Input anda salah, silahkan coba lagi"); continue;}
        else if (n == 5)
        {printf("Terimakasih, telah menggunakan kalkulator AULIA AZ ZAHRA"); break; }
    
        printf("Masukkan nilai pertama : ");
        scanf("%lf", &a);
        printf("Masukkan nilai kedua : ");
        scanf("%lf", &b);

        if (n == 1)
        {
            printf("Hasil penjumlahan antara %.2lf dengan %.2lf adalah %.2lf\n\n", a, b, a+b);
        }
        else if (n == 2)
        {
            printf("Hasil pengurangan antara %.2lf dengan %.2lf adalah %.2lf\n\n", a, b, a-b);
        }
        else if (n == 3)
        {
            printf("Hasil perkalian antara %.2lf dengan %.2lf adalah %.2lf\n\n", a, b, a*b);
        }
        else if (n == 4)
        {
            printf("Hasil pembagian antara %.2lf dengan %.2lf adalah %.2lf\n\n", a, b, a/b);
        }
    }
    return 0;
}